docs/hero-banner.md

```diff
@@ -17,13 +24,7 @@ Here are some cool activities to get you started with your @boardname@!
 * url: https://arcade.makecode.com/hour-of-code-2022
 * cardType: link
 ---
-* name: Holiday Bow Battle!
-* buttonLabel: Try Now
-* imageUrl: /static/hero-gallery/holiday.png
-* url: /tutorials/holiday
-* cardType: tutorial
----
-* name: How-to code a video game
+* name: How to Code a Video Game
```

Comment:
```suggestion
* name: How To Code a Video Game
```

Maybe? It looks like this one's contentious with AP being weird (https://titlecaseconverter.com/blog/is-to-capitalized/#:~:text=Another%20function%20of%20to%20is%20ito%20indicate), @ganicke do we follow a particular style guide for docs?